packages/config/config/devices/0x024f/fwc01.json

```
@@ -0,0 +1,227 @@
+{
+	"manufacturer": "Smartly AS",
+	"manufacturerId": "0x024f",
+	"label": "FWC01",
+	"description": "Wheel Controller",
+	"devices": [
+		{
+			"productType": "0x0003",
+			"productId": "0x1003"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "4",
+			"label": "Safety Activate Delay",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32767,
+			"defaultValue": 0
+		},
+		{
+			"#": "5",
+			"label": "Safety OFF Period",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32767,
+			"defaultValue": 600
+		},
+		{
+			"#": "6",
+			"label": "Safety ON Period",
+			"unit": "seconds",
+			"valueSize": 2,
+			"minValue": 0,
+			"maxValue": 32767,
+			"defaultValue": 300
+		},
+		{
+			"#": "10",
+			"label": "Dimming speed",
+			"description": "Dimmer Speed in 1/100th of a Second",
+			"unit": "/100s",
```

We usually do this (consistency)
```suggestion
			"unit": 0.01s",
```

---

Ok, I did not know this, so I'll fix it. I used an existing config template for a similar device from the same vendor as the base for this one. This was from there.